clang/include/clang/Driver/Options.td

```diff
@@ -1115,6 +1115,11 @@ defm cx_fortran_rules: BoolOptionWithoutMarshalling<"f", "cx-fortran-rules",
   NegFlag<SetFalse, [], [ClangOption, CC1Option], "Range reduction is disabled "
   "for complex arithmetic operations">>;
 
+def Wunknown_attribute_namespace_EQ : CommaJoined<["-"], "Wunknown-attribute-namespace=">,
+  Group<W_value_Group>, Flags<[HelpHidden]>, Visibility<[ClangOption, CC1Option]>,
+  HelpText<"Specify a comma-separated list of allowed unknown attribute namespaces">,
```

Comment:
It's a little bit odd that we have help text for something with the `HelpHidden` flag. Can probably remove the `HelpText`.

Reply:
@AaronBallman thanks for the feedback. There is a test that disallows such options from appearing in the help - should it be changed?

https://github.com/llvm/llvm-project/blob/ac94fade6075fec89eb29c7dedf01ef59601e61d/clang/test/Driver/immediate-options.c#L6